tracer/test/Datadog.Trace.IntegrationTests/DiagnosticListeners/AspNetCoreDiagnosticObserverTests.cs

```diff
@@ -143,6 +143,13 @@ public async Task DiagnosticObserver_ForRazorPages_SubmitsSpans(string path, Htt
             string childSpan2ResourceName,
             SerializableDictionary secondChildSpanTags)
         {
+#if NETCOREAPP2_1
```

Comment:
Can we check for .NET Core 2.1 at run time instead of compile time? I'd like to avoid any possible issues with the build target not being the same version as the runtime.

Reply:
never mind, we do this everywhere :P